includes/html/pages/device-dependencies.inc.php

```diff
@@ -29,12 +29,12 @@
     <table id="hostdeps" class="table table-hover table-condensed table-striped">
         <thead>
             <tr>
-                <th data-column-id="deviceid" data-visible="false" data-css-class="deviceid">No</th>
+                <th data-column-id="id" data-formatter="deviceid" data-type="int" data-sortable="true" data-visible="true">Id</th>
```

Comment:
Same here. No point in adding a formatter that does nothing

Reply:
And we don't want to show device id by default. It's not very useful info 99% of the times

Reply:
Sure, thanks for that.

Reply:
You should fix all comments in the same pull request

Reply:
> And we don't want to show device id by default. It's not very useful info 99% of the times

Actually, this "ID" is very important, for the Admin to aquire last ID inserted.
If we have many devices on librenms, and many people addin devices do monitor, the Admin needs to easily check those last devices inserted into librenms to check for configuration best-practices, as for example, if devices have been "type" updated, if Snmp "Port Association Mode" and max-repeaters are correct, If the Ports have been disabled, etc...